plugins/inappstore/in_app_store.mm

```diff
@@ -231,6 +231,15 @@ - (void)finishTransactionWithProductID:(NSString *)productID {
 	self.pendingTransactions[productID] = nil;
 }
 
+- (BOOL)paymentQueue:(SKPaymentQueue *)queue shouldAddStorePayment:(SKPayment *)payment forProduct:(SKProduct *)product {
```

Comment:
Oops...
Accidentally got my experience on a [deferred purchase.](https://developer.apple.com/documentation/storekit/skpaymenttransactionobserver/2877502-paymentqueue?language=objc)

The event is used when a purchase has been initiated outside of the app.